components/browser/storage-sync/src/main/java/mozilla/components/browser/storage/sync/RemoteTabsStorage.kt

```diff
@@ -52,13 +48,12 @@ open class RemoteTabsStorage : Storage, SyncableStore {
      */
     suspend fun getAll(): Map<SyncClient, List<Tab>> {
         return withContext(scope.coroutineContext) {
-            val allTabs = api.getAll() ?: return@withContext emptyMap<SyncClient, List<Tab>>()
-            allTabs.map { device ->
-                val tabs = device.tabs.map { tab ->
+            api.getAll().map { device ->
```

Comment:
Yes uniffi will now guarantee an empty list via the [lower level rust api in the a-s commit](https://github.com/mozilla/application-services/pull/4192/files#diff-aa70476497fdc778137aa9273f53c6bebc812a608fb8de6d3523adf330dbe325R58)

Where before it could've been null mentioned [in the removed code here](https://github.com/mozilla/application-services/pull/4192/files?file-filters%5B%5D=.kt&file-filters%5B%5D=.proto&file-filters%5B%5D=.rs&file-filters%5B%5D=.toml&file-filters%5B%5D=.udl#diff-2a772527e8d9d9fbc9317a725be43f0bb522ee5bbb96e87fca944ee396e653f6L41) so I _think_ that was a limitation of how it used to be pass over the ffi